content/presentations/conf-sce/index.md

```diff
@@ -0,0 +1,95 @@
+---
+title: Conférence du club "Comment réduire son empreinte numérique"
+summary: "Semaine Climat Énergie 2022-20223"
+date: 2023-01-20 
```

Comment:
Tu peux mettre la date pour le 25.
La page ne sera pas build →[Draft, future, and expired content](https://gohugo.io/getting-started/usage/#draft-future-and-expired-content)

Reply:
Le pb c'est que du coup elle n'apparaîtra pas avant et l'idée est qu'elle apparaisse tout de suite.

Reply:
Tu peux build avec -F et la page apparait